config/chewy.rb

```diff
@@ -0,0 +1,25 @@
+Chewy.logger = Rails.logger
+
```

Comment:
@tarebyte - Did you evaluate using other Chewy strategies?

I generally prefer using an async Job strategy to get the ind updating time out of the request. It's as simple as adding this. Also, the `chewy` job type should added to the sidekiq entry in `Procfile` with this.

```ruby
# Use ActiveJob config for async index updates.
Chewy.strategy(:active_job)

# Chewy wraps controller actions in the atomic strategy by default.
# We change this to use ActiveJob here. Makes index updates async, outside
# of the request.
Chewy.request_strategy = :active_job
```